### DISCUSSION TITLE
Tasmota 14.0.0 after update from 13.4.0 doesn't read MQTT commands from Domoticz

### DISCUSSION THREAD

**Discussion by SkowronK** [arendst/Tasmota#21476] [Support request] [answered]
Hi after update to 14.0 Tasmota stopped reacting to changing actuator states in Domoticz. (communication via MQTT).
You cannot toggle state from Domoticz however toggling state in Tasmota is visible in Domoticz.

Devices used: sonoff mini, sonoff 4CH, gosunds EP2 so it seems to be not related to hardware.
Tasmota slashed with standard (release-tasmota) image.

Downgrading to 13.4 returns full functionality.

Anyone experienced similar behaviour?

K.

**Comment by arendst** [comment]
First chk dev changelog for a possible solution. Then try dev and if still fails continue here.

**Reply by HarryHase4** [reply]
I can confirm the fix (tested with 14.0.0.0 and 14.0.0.4)

**Reply by ilexpl01** [reply]
Me two. Installed 14.0.0.4 on 20 sooff devices and no further issues with Domoticz control.

**Comment by arendst** [comment, marked as answer]
Fixed in v14.0.0.1
